contracts/credit/CreditManagerV3.sol

```diff
@@ -716,15 +712,15 @@ contract CreditManagerV3 is ICreditManagerV3, SanityCheckTrait, ReentrancyGuardT
         }); // U:[CM-21]
         cdd.cumulativeQuotaInterest += currentCreditAccountInfo.cumulativeQuotaInterest - 1; // U:[CM-21]
 
-        cdd.accruedInterest = cdd.cumulativeQuotaInterest;
-        cdd.accruedInterest += CreditLogic.calcAccruedInterest({
+        cdd.accruedInterest = CreditLogic.calcAccruedInterest({
             amount: cdd.debt,
             cumulativeIndexLastUpdate: cdd.cumulativeIndexLastUpdate,
             cumulativeIndexNow: cdd.cumulativeIndexNow
-        }); // U:[CM-21]
+        });
+        cdd.accruedFees = currentCreditAccountInfo.quotaFees + cdd.accruedInterest * feeInterest / PERCENTAGE_FACTOR;
```

Comment:
Consider using the `mulDiv` function or preventing phantom overflow in some other way.